flipperdevices/Flipper-Android-App
# Single tap infrared

**Background**

When pressing infrared buttons anywhere - the signal will be send not once. FW added new request which allows one-time infrared signal send

**Changes**

- Change EmulateApi with new one-time send

**Test plan**

- Build firmware from https://github.com/flipperdevices/flipperzero-firmware/pull/4000
- Open flipper app
- Open some infrared key
- Press key once and see emulating on flipper now shorter
- Repeat this step for Infrared Remote Controls
